src/Table/Synapse/TableIndexDefinition.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Table\Synapse;
+
+/**
+ * @todo this class is missing validation for CLUSTERED INDEX now, but this is not yet used
```

Comment:
to imho není pravda. Ona se ta validace provádí, ale ta metoda je prázdná, tzn by to todo mělo být spíše u té metody

Reply:
Přidal sem a použil pro vypsání nové AVAILABLE konstanty ^